db/schema.rb

```diff
@@ -906,6 +916,7 @@
   add_foreign_key "commontator_subscriptions", "commontator_threads", column: "thread_id", on_update: :cascade, on_delete: :cascade
   add_foreign_key "course_self_joins", "courses"
   add_foreign_key "divisions", "programs"
+  add_foreign_key "feedbacks", "users", on_delete: :cascade
```

Comment:
Technically, that is certainly a correct solution. I am just a little worried that we are now using different methods that achieve the same in our `user` model and mix them. I did not count it, but in `user.rb` you will find lots of relations that have the `dependent: :destroy` flag set. So these are all at the same place. By delegating the cascading for `feedbacks` to the database, we now open up a second place where behavior for such dependencies is stored. Maybe there should at least be a comment in user.rb that there is also a relation to feedbacks, but that deletion is handled on a database levels (alternatively, add `has_many :feedbacks, dependent: :destroy` to `user.rb`, then the cascading in the migration is not necessary). Note that there a lot of articles in the net about `dependent: :destroy` vs '`on_delete: :cascade)`'.

Reply:
Oh good point. I agree we should respect business logic when deleting an active record and have the additional Rails layer, i.e. favor `has_many :feedbacks over dependent: :destroy` with the latter working directly on the database. I've just changed it in 07ba509b828c.